frontend/javascripts/oxalis/view/left-border-tabs/modals/add_volume_layer_modal.tsx

```diff
@@ -84,14 +84,18 @@ export default function AddVolumeLayerModal({
   dataset,
   onCancel,
   tracing,
+  preselectedLayerIndex,
+  showLayerSelectionDisabled,
 }: {
   dataset: APIDataset;
   onCancel: () => void;
   tracing: Tracing;
+  preselectedLayerIndex: number | null | undefined;
+  showLayerSelectionDisabled: boolean | null | undefined;
```

Comment:
```suggestion
  preselectedLayerIndex: number | undefined;
  showLayerSelectionDisabled: boolean | undefined;
```

should be enough. the reason the `| null | undefined` is scattered through the entire code base is the automatic flow-to-typescript conversion and subtle differences between the semantics of their syntax. in the longterm, I'd rather get rid of this :)